templates/desktop/developers.html

```diff
@@ -443,200 +443,200 @@ <h2>Developers love Ubuntu, organisations trust it</h2>
     <div class="row">
       <div class="col-6">
         <p class="u-no-margin--bottom">
-          <strong>Operating systems developers prefer</strong>
-          <br aria-hidden />
-          Multiple choice poll
+          <strong>Which of these open source technologies
+            does your organization use today to support
+            your software infrastructure?
+          </strong>
         </p>
         <div id="hackerearth-chart" class="hackerearth-chart u-hide--small"></div>
         <div class="u-hide--medium u-hide--large">{% include "desktop/partial/_hackearth-table.html" %}</div>
 
         <div class="u-hide--large">
           <hr class="p-rule--muted" />
           <p>
-            Source: <a href="https://media.hackerearth.com/recruit/wp-content/uploads/2020/05/he-developer-survey-2020.pdf">The 2020
-          HackerEarth Developer Survey</a>
+            Source: <a href="https://www.openlogic.com/resources/state-of-open-source-report">2025 State of Open Source Report, Perforce OpenLogic</a>
+          </p>
+        </div>
+      </div>
+      <div class="col-6">
+        <p class="u-no-margin--bottom">
+          <strong>Operating systems IoT developers prefer –
+            Multiple choice poll
+          </strong>
```

Comment:
The `Multiple choice poll` text is not bold in the copy doc (or the existing implementation)

```suggestion
          <strong>Operating systems IoT developers prefer –
          </strong>
            Multiple choice poll
```

Copy doc:
<img width="560" height="182" alt="Image" src="https://github.com/user-attachments/assets/6a6a1f18-4dc2-4a02-b0ea-25cc89b5b1ea" />

Existing:
<img width="364" height="209" alt="Image" src="https://github.com/user-attachments/assets/5257f934-46a1-4337-93bd-54f365213a08" />

Reply:
@fasih-mehmood fixed the bug from screenshot.

The word choice is up to the copydoc which you can find [here](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit?tab=t.0)

If the current document is in sync with that, we don't need to change it.

Reply:
The word choice is indeed in sync with the copydoc. We've started adopting the US spelling across all of the pages; The newly suggested title also uses US spelling, whereas all the other instances in the same page still use the UK spelling.

<img width="434" height="129" alt="image" src="https://github.com/user-attachments/assets/baa7961b-e3b0-4894-8e77-28c12f8a3660" />

But it's fine, we can leave it for when the PMs actually request the change.